registered_span.go

```diff
@@ -1054,3 +1064,55 @@ func NewAWSDynamoDBSpanTags(span *spanS) AWSDynamoDBSpanTags {
 
 	return tags
 }
+
+// AWSInvokeSpanTags contains fields within the `aws.lambda.invoke` section of an OT span document
+type AWSInvokeSpanTags struct {
+	// FunctionName is a name of the function which is invoked
+	FunctionName string `json:"function"`
+	// InvocationType if equal to `Event`, means it is an async invocation
```

Comment:
What are the other possible values here? If it's only to distinguish between sync/async, should we use a boolean instead?

Reply:
There are three options (Event | RequestResponse | DryRun). These names are from the AWS documentation and they are shown in UI. This why I think it makes sense to have a string type here.